benchmarks/graphics_pipeline/main.cpp

```diff
@@ -799,6 +809,15 @@ void ProjApp::Render()
     // Wait for and reset render complete fence
     PPX_CHECKED_CALL(frame.renderCompleteFence->WaitAndReset());
 
+    // Read query results
+    if (GetFrameCount() > 0) {
+        uint64_t data[2] = {0, 0};
+        PPX_CHECKED_CALL(frame.timestampQuery->GetData(data, 2 * sizeof(uint64_t)));
```

Comment:
`sizeof(data)`, compiler knows its length, so you'll get `sizeof(data[0])*2`

Reply:
Done.